index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Concert-website</title>
+    <link rel="stylesheet" href="./main.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+
+</head>
+
+<body>
+    <section class="top-nav">
+        <div></div>
+        <div class="top-nav-content">
+            <img src="./items/facebook-icon.png" alt="Facebook icon">
+            <img src="./items/twitter-icon.png" alt="twiter icon">
+            <a href="">English</a>
+            <a href="">My Page</a>
+            <a href="">Logout</a>
+        </div>
```

Comment:
- [ ] Kindly add some margin around each element for this section as to match the given design

Your design
![image](https://user-images.githubusercontent.com/81932478/196939634-f0db0c9a-b65e-44c6-9441-84e00d05c915.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/196939958-eba43df1-5a3a-4681-aed6-1c96a5d3c9a1.png)